workers/data_refinery_workers/processors/smasher.py

```diff
@@ -264,18 +267,30 @@ def _write_tsv(job_context, metadata, smash_path):
         for species in job_context['input_files'].keys():
             species_dir = smash_path + species + '/'
             os.makedirs(species_dir, exist_ok=True)
-            with open(species_dir + species + '_metadata.tsv', 'w') as output_file:
-                dw = csv.DictWriter(output_file, columns, delimiter='\t')
+            samples_in_species = []
+            with open(species_dir + "metadata_" + species + '.tsv', 'w') as tsv_file:
+                dw = csv.DictWriter(tsv_file, columns, delimiter='\t')
                 dw.writeheader()
                 for sample_metadata in metadata['samples'].values():
                     if sample_metadata.get('refinebio_organism', '') == species:
                         row_data = _get_tsv_row_data(sample_metadata)
                         dw.writerow(row_data)
+                        samples_in_species.append(sample_metadata)
+
+            # Writes a json file for current species:
+            if len(samples_in_species):
+                species_metadata = {
+                    'species': species,
+                    'samples': samples_in_species,
+                    'created_at': datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%S')
```

Comment:
I'm not sure about this keyvalue, as it it will look like that's when the samples were created, not the metadata, and it will change slightly throughout the loop. I think we can remove it unless it's been specifically asked for?

Reply:
That's a good point. I added it mainly because it was added in the json file that includes all samples:
https://github.com/AlexsLemonade/refinebio/blob/cbef527aca99f9d30efd8ebd5d8aa739d1002d46/workers/data_refinery_workers/processors/smasher.py#L612
What do you think @dvenprasad?

Reply:
:+1: